tests/test_doctestplus.py

```diff
@@ -435,6 +435,29 @@ def test_requires(testdir):
     testdir.inline_run(p, '--doctest-plus', '--doctest-rst').assertoutcome(skipped=1)
 
 
+def test_requires_all(testdir):
+    testdir.makeini(
+        """
+        [pytest]
+        doctestplus = enabled
+    """)
+
+    # should be ignored
+    p = testdir.makefile(
+        '.rst',
+        """
+        .. doctest-requires-all:: foobar
+
+            >>> import foobar
+
+        This is a narrative line, before another doctest snippet
+
+           >>> import foobar
```

Comment:
Fix formatting to reflect typical real use case.
```suggestion
        This is a narrative line, before another doctest snippet::

            >>> import foobar
```

Reply:
This should not matter, all the snippets should be collected no matter if there are `::` or correct number of spaces before the `>>>`.